### DISCUSSION TITLE
does host_allow_list work in opensource?

### DISCUSSION THREAD

**Discussion by tooptoop4** [grafana/grafana#73867] [Alerting]
https://grafana.com/docs/grafana/latest/setup-grafana/configure-security/configure-request-security/ mentions open source at the top but then later says enterprise? I couldn't see it in the code either

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82935. Feel free to continue the discussion around this there. Thank you!
